decidim-admin/app/views/decidim/admin/participatory_processes/show.html.erb

```diff
@@ -30,11 +30,22 @@
     :slug,
     :short_description,
     :description
-    %>
+  %>
+
   <dt><%= display_label(@participatory_process, :hero_image) %></dt>
-  <dd><%= image_tag @participatory_process.hero_image.url.to_s %></dd>
+  <dd>
+    <% if @participatory_process.hero_image %>
+      <%= image_tag @participatory_process.hero_image.url.to_s %>
```

Comment:
Does this really fail when no `hero_image` is set? As far as I remember, `hero_image` will always return a Carrierwave object that will respond to `.url`. Calling `url.to_s` avoids calling `image_tag(nil)`, which breaks, as `nil.to_s` returns an empty string.